rita_common/src/network_endpoints/mod.rs

```diff
@@ -45,6 +27,26 @@ pub async fn make_payments(item: Json<PaymentTx>) -> HttpResponse {
     HttpResponse::Ok().json("Payment Received!")
 }
 
+/// The recieve side of the make payments v2 call. This processes a list of payments instead of a single payment
+pub async fn make_payments_v2(item: Json<HashSet<PaymentTx>>) -> HttpResponse {
+    let pmt_list = item.into_inner();
+
+    for pmt in pmt_list {
+        let local_payment_list = get_payment_txids(pmt.from, false);
+
+        if !local_payment_list.contains(&pmt) {
```

Comment:
I don't see a reason to compare to the local payments list, we already check that we don't double validate a payment in the payment validator, so this is a duplicate.

Reply:
If your talking about the chekc in validate_later that used to see 
If pmt in successful.transactions {
dont add to unvalidated.trasactions
}
I removed that check and moved it here since it was a little more compliacted to write that with new new hashmap<id -> hashset> format of datastore

If youre talking about payment validator not adding it to successfull transactions since we use a hashmap at the end, why would we want to send payment validator a bunch of tx that we know we already accounted for?

Reply:
Sending payment validator a bunch of tx we know are accounted for is fine so we can de duplicate the process of checking

Payment validator can validate millions of tx. We don't really have to worry about putting more load on it. We do have to worry about code or logic errors and the more places we do different things and the more spread out the validation logic is the more likely it is that we will make a mistake, which can have expensive consequences.